go.mod

```diff
@@ -2,23 +2,21 @@ module maistra.io/api
 
 go 1.15
 
-replace github.com/maistra/xns-informer => github.com/luksa/xns-informer v0.0.0-20220204100742-3436f0ac2527
-
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/go-cmp v0.5.6
-	github.com/maistra/xns-informer v0.0.0-20210707160032-977ec17e2e0e
+	github.com/maistra/xns-informer v0.0.0-20220823123640-04d03c2c082f
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
 	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.22.2
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
-	k8s.io/code-generator v0.22.2
-	sigs.k8s.io/controller-runtime v0.9.6
-	sigs.k8s.io/controller-tools v0.6.2
+	k8s.io/api v0.23.5
+	k8s.io/apiextensions-apiserver v0.22.4
```

Comment:
I wonder if it makes sense to align those versions with what we have in istio 2.3: https://github.com/maistra/istio/blob/maistra-2.3/go.mod

(controller-runtime below too)